backends/rapidpro/status.go

```diff
@@ -284,7 +285,7 @@ FROM
 AS 
 	s(msg_id, channel_id, status, external_id) 
 WHERE 
-	msgs_msg.id = s.msg_id::int AND
+	msgs_msg.id = s.msg_id::bigint AND
```

Comment:
what happens without these casts?

Reply:
Doesn't work sadly.. going to investigate if there's another way though.